package.json

```diff
@@ -39,7 +39,7 @@
     "dev": "turbo dev --filter=!./playgrounds/*",
     "test": "cargo test && vitest run --hideSkippedTests",
     "test:integrations": "vitest --root=./integrations",
-    "test:ui": "pnpm run --filter=tailwindcss test:ui",
+    "test:ui": "pnpm run --filter=tailwindcss test:ui && pnpm run --filter=@tailwindcss/browser test:ui",
```

Comment:
Can/should we merge these? I think we can use `--filter` twice 🤔

Reply:
If you do that it does some super super weird stuff to the console output

Reply:
Just tried it, that is … interesting 😂